internal/pb_agent/middleware.pb.go

```diff
@@ -7,10 +7,11 @@
 package pb_agent
 
 import (
-	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
-	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
 	reflect "reflect"
 	sync "sync"
+
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
```

Comment:
This is an odd change - is our lint test not catching non-gofmt'd changes?

Reply:
This was correctly gofmt'd before, but not goimport'd.

We can update the lint separately to enforce that if we want to have something more strict than just gofmt.
I just ran goimports to group the new import path more consistently, and it happened to also pick this up.